minetest/minetest
# Better F6 profiler

## Goal
 * Make the profiler easier to read
 * Rename profilers to they make more sense
 * Drop useless profilers or merge with others

## Result
 * Lower default update interval to 2 seconds
 * Show `ScopeProfiler` results in milliseconds
 * Many removed variables used for time measuring, or such that did not do anything

![screenshot_20190804_172329](https://user-images.githubusercontent.com/1497498/62425855-3b931280-b6de-11e9-8d5a-b3d26a78aee7.png)

## To do

This PR is Ready for Review.

## How to test

1) Join a world
2) Same in-game performance (you'd notice that if there were accidental changes in the code)
3) Open the F6 profiler -> monospace text, nothing's cut off
4) Open the F5 graphs -> must all display values when walking around
